Source/Scene/Scene.js

```diff
@@ -2468,7 +2468,7 @@ define([
             scene._fxaa.clear(context, passState, clearColor);
         }
 
-        if (environmentState.isSunVisible && scene.sunBloom && !useWebVR) {
+        if (environmentState.isSunVisible && scene.sunBloom && !useWebVR && !picking) {
```

Comment:
And, notably, the offending commit didn't touch the line I've changed here.  It remotely changed the results of this `if` check.